exporter/exporter.go

```diff
@@ -25,6 +25,10 @@ type TraceExporter interface {
 
 	// Name gets the name of the trace exporter.
 	Name() string
+
+	// Stop is invoked to stop the exporter and do any necessary clean up.
+	// Typically this is invoked during service shutdown.
+	Stop() error
```

Comment:
As @bogdandrutu suggested let's name this Shutdown to make it consitent with Exporter interface defined in OpenTelemetry spec (technically the Exporter in the Service and the Exporter in the spec are not the same thing, but it beneficial to use to consistent terminology across the project).

See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-exporter.md

Reply:
I just posted a comment in the issue. Currently the receivers have a stop method so to be consistent with that component I think it is best to keep it as stop.
https://github.com/open-telemetry/opentelemetry-service/issues/246#issuecomment-521393640
Within the service, the toplevel fn is shutdown and it invokes stop for receivers and exporters.  

Alternatively, I can create an issue for someone to update the receivers to use the term shutdown.

Reply:
Please, create the issue to rename those.

Reply:
Issue #261 created. I'll update the name to shutdown.